piglow/examples/strobe.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"time"
+
+	"github.com/goiot/devices/piglow"
+	"golang.org/x/exp/io/i2c"
+)
+
+func main() {
+	p, err := piglow.Open(&i2c.Devfs{Dev: "/dev/i2c-1", Addr: 0x54})
+	if err != nil {
+		panic(err)
+	}
+
+	defer p.Close()
+
+	if err := p.Reset(); err != nil {
+		panic(err)
+	}
+
+	for {
```

Comment:
Would a control+C still properly trigger the defer statement? If that's the case, then yes we could just use extend the defer statement and make it reset and close without using signals. The reason why I think it's important to show real like examples here is because new users need a way to get started. Part of that means showing a proper/recommended way to clean off state.
In this example: https://github.com/goiot/devices/blob/master/grove/accel3xdigital/examples/monitor/main.go I used signals and it does add more complexity. If we prefer to use defer, then we should use that in all these examples.

Note also that simple copy/pasteable examples should live in the example tests, files in the examples folder can be more complex and show off more features.